docs/api/cozy-stack-client.md

```diff
@@ -1690,8 +1706,9 @@ Returns true when parameter has type directory, file or has _type io.cozy.files
 
 | Param | Type | Description |
 | --- | --- | --- |
-| type | <code>string</code> | The type of the file |
-| _type | <code>string</code> | The _type of the file |
+| doc | <code>object</code> | The document whose type is checked |
+| [doc._type] | <code>string</code> | The document's doctype |
+| [doc.type] | <code>&#x27;directory&#x27;</code> \| <code>&#x27;file&#x27;</code> | The io.cozy-files document type |
```

Comment:
&#x27;file&#x27;

Reply:
I'm referring to the `io.cozy.files` doctype here, not its `type` value which can either be `file` or `directory`.